easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'CmdCp'
+
+name = 'Pindel'
+version = '0.2.5a7'
```

Comment:
(!)

I stand corrected!

On 3 June 2015 at 20:04, Pablo Escobar Lopez <notifications@github.com>
wrote:

> 0.2.5a7 is the latest release https://github.com/genome/pindel/releases
>


-- 


echo "sysadmin know better bash than english"|sed s/min/mins/ \
  | sed 's/better bash/bash better/' # signal detected in a CERN forum